commands/validate.go

```diff
@@ -0,0 +1,91 @@
+package commands
+
+import (
+	"fmt"
+	"github.com/Masterminds/semver"
+	"github.com/pivotal-cf/jhanda"
+	"github.com/pivotal-cf/kiln/internal/cargo"
+	"gopkg.in/src-d/go-billy.v4"
+	"gopkg.in/src-d/go-billy.v4/osfs"
+	"strings"
+)
+
+type Validate struct {
+	Options struct {
+		Kilnfile       string   `short:"kf" long:"kilnfile" default:"Kilnfile" description:"path to Kilnfile"`
```

Comment:
**suggestion:** I like examples in my help descriptions.

Reply:
Maybe we can create an issue for this so we do it across all the flags.